stix_shifter_modules/aws_athena/stix_translation/README.md

```diff
@@ -0,0 +1,422 @@
+```
+
+### Exclusions 
+Athena does not provide operator support for the following attributes,
+
+`LIKE Operator`
+
+* 'LIKE' Operator is not supported for vpcflow protocol field with wildcard characters.
+
+`MATCHES Operator`
+
+* 'MATCHES' Operator is not supported for vpcflow protocol field with regular expressions in the search query.
+
+### STIX pattern for custom attributes and sample values
+|  Description  |  STIX Pattern  |  Sample Values  |
+| --- | --- | --- |
+| Network Interface id of the EC2 instance  | ipv4-addr:x_aws_interface_id | [ipv4-addr:x_aws_interface_id = 'eni-0a70b0fa1a9cd3dbe'] |
+| City name of Remote Ip address  | ipv4-addr:x_aws_remote_city_name | [ipv4-addr:x_aws_remote_city_name = 'Ashburn'] |
+| Country name of Remote Ip address  | ipv4-addr:x_aws_remote_country_name | [ipv4-addr:x_aws_remote_country_name = 'United States'] |
+| Network Interface id of the EC2 instance  | ipv6-addr:x_aws_interface_id | [ipv6-addr:x_aws_interface_id = 'eni-0a70b0fa1a9cd3dbe'] |
+| AWS Account Id | x-aws-details:account_id | [x-aws:account_id = '979326520502'] |
+| AWS Region | x-aws-details:aws_region | [x-aws:aws_region = 'us-east-1'] |
+| EC2 instance Id  | x-aws-instance:instance_id | [x-aws-instance:instance_id = 'i-091501e21e01d0602'] |
+| EC2 instance Image Id  | x-aws-instance:image_id| [x-aws-instance:image_id = 'ami-00068cd7555f543d5'] |
+| EC2 instance Availability Zone  | x-aws-instance:availability_zone | [x-aws-instance:availability_zone = 'us-east-1a'] |
+| VPC Id associated with EC2 Instance | x-aws-vpc:vpc_id | [x-aws-vpc:vpc_id = 'i-091501e21e01d0602'] |
+| Subnet Id associated with EC2 Instance | x-aws-vpc:subnet_id| [x-aws-vpc:subnet_id = 'ami-00068cd7555f543d5'] |
+| Security Group Name associated with EC2 Instance  | x-aws-vpc:security_group_name | [x-aws-vpc:security_group_name = 'launch-wizard-1'] |
+| Security Group Id associated with EC2 Instance | x-aws-vpc:security_group_id | [x-aws-vpc:security_group_id = 'sg-0aa89ff4646f71594'] |
+| AccessKey Id of User | x-aws-api:access_key_id | [x-aws-api:access_key_id = 'AAAABBBBCCCC'] |
+| AWS api name | x-aws-api:api | [x-aws-api:api = 'DescribeSecurityGroups'] |
+| AWS service name whose api is invoked | x-aws-api:api_service_name | [x-aws-api:api_service_name = 'ec2.amazonaws.com'] |
+| Name of ibm finding | x-ibm-finding:name | [x-ibm-finding:name = '146.168.246.36 is performing SSH brute force attacks against i-03d7e6195920aa4c0.'] |
+| Id of ibm finding | x-ibm-finding:finding_id | [x-ibm-finding:finding_id = '14b76d5936d5f302695e67ac500ab78a'] |
```

Comment:
Remove the finding id.

Reply:
@delliott90 The finding id is now removed from the README samples and the file is updated in PR branch